testing-framework/hetzner/network/variables.tf

```diff
@@ -43,17 +38,18 @@ variable "additional_node_ips" {
     bootstrap = [""]
     node      = [""]
     farmer    = [""]
+    domain    = [""]
   }
 }
 
 variable "ssh_key_name" {
-  default = "hetzner"
   type    = string
+  default = "hetzner"
```

Comment:
I dont understand what the default here is

Reply:
it's the key name, this is used with AWS but not needed for hetzner, it has been removed.